text/0000-pack-buildpack-new-alternatives.md

```diff
@@ -0,0 +1,275 @@
+? Choose the buildpack API version (use the default if you are unsure):
+    0.7
+  ▸ 0.8
+✔ 0.8
+Enter an identifier for the buildpack: example/golang
+Enter a stack this buildpack will support by default: io.buildpacks.samples.stacks.bionic
+✔ Enter a valid Go module name for this buildpack: github.com/user/buildpack
+Created project in go_buildpack
+```
+
+A user may specify the choice of template from a template collection via a `--sub-path` flag.  A user may choose a `bash` template from a template collection and provide the api, output directory and stacks as command line flags:
+
+```command
+$ pack buildpack create --template https://github.com/AidanDelaney/cnb-buildpack-templates \
+  --sub-path Go \
+  --arg BuildpackID=example/golang \
+  --arg BuildpackAPI=0.8 \
+  --arg ProjectDirectory=go_buildpack \
+  --arg BuildpackStacks=io.buildpacks.samples.stacks.bionic
+Created project in go_buildpack
+```
+
+Having provided examples of the use of `pack buildpack create`.  We now specify the format of `prompts.toml`.
+
+## prompts.toml
+
+The prompts are contained in a TOML file.  The `prompts.toml` file contains an array of tables.  Each table specifies a single `prompt`.  Each prompt defines a variable identified by a provided `name`.
+
+| field         | required | description                                                                                                                              | type           |
+|---------------|----------|------------------------------------------------------------------------------------------------------------------------------------------|----------------|
+| name          | Yes      | The variable identifier.  Must be unique within the `prompts.toml` file.                                                                 | string         |
+| prompt        | Yes      | The default prompt question to ask the user                                                                                              | string         |
+| prompt.locale | No       | A translation of the default prompt into the locale specified.                                                                           |                |
+| required      | No       | Evaluates to `false` if not provided.  Specifies whether the user **must** provide a value for the variable identified by `name`.        | boolean        |
+| default       | No       | A default value for the variable identified by `name`.  Mutually exclusive to `choices`                                                  | string         |
+| choices       | No       | A list of default values from which the user may choose a value for the variable identified by `name`.  Mutually exclusive to `default`. | list of stings |
+
+Prompt locales are specified following [IETF BCP 47](https://tools.ietf.org/rfc/bcp/bcp47.txt).  As such `prompt.en-US` is the translation of `prompt` into US English and `prompt.es-MX` is the translation of `prompt` into Mexican Spanish.
```

Comment:
1. Is there an anticipated use for locale or should we defer this added complexity until the need arises?
2. Can you give a concrete example of how locales would work? How is locale determined at execution time?

Reply:
I mostly didn't want to assume that the world is centered around a single language.  I've now removed this from the RFC, but I think it's something that I'd like to support in the medium term.